mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -230,7 +230,7 @@ class Word2VecModel private[ml] (
    * Find "num" number of words closest in similarity to the given word, not
    * including the word itself.
    * @return a dataframe with columns "word" and "similarity" of the word and the cosine
-   * similarities between the synonyms and the given word vector.
+   * similarities between the synonyms and the given word string.
```

Comment:
You can just say "given word" if you're going to change the comment - drop the "string"